### DISCUSSION TITLE
Change Taxonomy Slug

### DISCUSSION THREAD

**Discussion by mundgold** [statamic/cms#4876] [Q&A] [answered]
Hello,

what do I have to change, if I want an other taxonomy-slug ?

So far I created a taxonomy "categories" I added it to a collection "series" 
all entries of series references to "categories"

My entries of specific category reached via {base_url}/categories/{term-slug}

Now I want to change that to {base_ur}/***kategorien***/{term-slug}

what do I have to change now ?

Thank you for your help!
Roland

**Comment by edalzell** [comment, marked as answer]
You'll have to change it in the files. Rename the taxonomy yaml file.

**Reply by mundgold** [reply]
Thank you for your reply!

... that means, I have to change every single collection-item ...

for example:

```
---
id: 2b416e07-addb-4ce5-9339-6a3123fab65f
blueprint: series
reihen_id: '5005264'
title: 'Achtsam morden'
autor: 'Karsten Dusse'
updated_by: beebe184-cb59-4976-b387-374b9604f671
updated_at: 1639054230
categories:
  - spannung
  - bestseller
---

```
to 
```
---
id: 2b416e07-addb-4ce5-9339-6a3123fab65f
blueprint: series
reihen_id: '5005264'
title: 'Achtsam morden'
autor: 'Karsten Dusse'
updated_by: beebe184-cb59-4976-b387-374b9604f671
updated_at: 1639054230
kategorien:
  - spannung
  - bestseller
---

```
and I have to rename 
````
taxonomies/categories/

to 

taxonomies/kategorien/

and

taxonomies/categories.yaml

to

taxonomies/kategorien.yaml

and also the

views/categories/

to 

views/kategorien/

Is that correct ?  ... or is there a shorter way ?

Thank you 
Roland

**Reply by edalzell** [reply]
yup

**Reply by mundgold** [reply]
Thank you ...
I should think twice about how to name a taxonomy ...

**Comment by Paperweb** [comment]
I had to find the hard way what to do when changing a taxonomy slug. 
It is "non-trivial'' indeed.

We have to do a little bit more than changing files, though, especially if collections have been assigned with it.

Here is what needs to be changed for a new {taxonomy} slug :

- content/taxonomies/{taxonomy.yaml}
- content/taxonomies/{taxonomy}/
- content/taxonomies/{taxonomy}/{term.yaml}
- content/collections/{collection-assigned-with-taxonomy.yaml}
- content/collections/{collection-assigned-with-taxonomy}/{entry-assigned-with-taxonomy-term.md}
- resources/blueprints/taxonomies/{taxonomy}/
- resources/blueprints/taxonomies/{taxonomy}/{taxonomy.yaml}

Then, in the control panel, we need to :
- edit and save the taxonomy
- edit and save the entries assigned to the taxonomy

Hope it clarifies further for anyone stumbling here!
